<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Игра Виселица</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string>View1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="questionLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QGraphicsView" name="graphicsView">
             <property name="minimumSize">
              <size>
               <width>266</width>
               <height>375</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-style: outset;</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-style: outset;</string>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QGroupBox" name="answerBox">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>70</height>
                 </size>
                </property>
                <property name="title">
                 <string/>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_3"/>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="lettersBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="lettersLayout"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab2">
       <attribute name="title">
        <string>View2</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>60</height>
              </size>
             </property>
             <property name="title">
              <string>Вопрос</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="questionLabel2">
                <property name="text">
                 <string>вопрос</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="answerBox2">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>35</height>
                 </size>
                </property>
                <property name="title">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="usedLattersBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>60</height>
              </size>
             </property>
             <property name="title">
              <string>Использованные буквы</string>
             </property>
             <layout class="QHBoxLayout" name="usedLettersLayout"/>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>60</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="title">
              <string/>
             </property>
             <widget class="QLabel" name="label">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>111</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>Осталось попыток:</string>
              </property>
             </widget>
             <widget class="QLabel" name="attemptsNumber">
              <property name="geometry">
               <rect>
                <x>120</x>
                <y>10</y>
                <width>49</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="allLetters">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="lettersLayout2"/>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="View.qrc"/>
 </resources>
 <connections/>
</ui>
